freqtrade/leverage/liquidation_price.py

```diff
@@ -0,0 +1,357 @@
+from typing import Optional
```

Comment:
The way this file is looking, and considering from where we're calling it (`freqtradebot.py`) makes me wonder if we could move these methods to each exchange class instead of having a case-statement here for each exchange.

That way, the logic per exchange would remain in one place.

This wasn't possible for interest as we'd cause a cyclic import (it's used from `models.py`) - but i don't see this here (the only other place this might need to be used is backtstesting - where the exchange is also available.)

That might even simplify the interface - as we can retrieve certain information (like the brackets) from within the method, instead of providing it as parameter.